address_book/Makefile

```diff
@@ -13,3 +13,19 @@ gen-proto: docker-build-generator
 		--grpc-gateway_out=. --grpc-gateway_opt=logtostderr=true --grpc-gateway_opt=paths=source_relative --grpc-gateway_opt=generate_unbound_methods=true \
 		-I /usr/local/include/. \
     	-I /api/. api.proto
+
+.PHONY: run-db
```

Comment:
Additional Makefile target to run docker-compose deployment is good to have.

Reply:
Done